### PR TITLE
k8s: add intro blog post

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

**Comment by hunchback** [comment]
@lebauce - happy if you could review this

File: blog/index.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ title: Blog
 layout: blog-index
 ---
 
+# [Introducing the K8s Probe](/blog/k8s-probe.html)
```
Comment (lebauce) [inline blog/index.md]: s/K8s/Kubernetes/

File: blog/index.md [hunk 2]
```diff
@@ -3,6 +3,14 @@ title: Blog
...
+## by Aidan Shribman, 01/05/2019
+
+In this article I introduce the K8s Probe which constructs the topological view
```
Comment (lebauce) [inline blog/index.md]: s/K8s Probe/Kubernetes probe/

File: blog/k8s-probe.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
+---
+title: Introducing the Kubernetes Probe
+layout: blog-post
+author: Aidan Shribman
+date: 24/04/2019
+---
+
+Since Skydive v0.22.0 the K8s topology probe is production ready. In this blog
```
Comment (lebauce) [inline blog/k8s-probe.md]: It probably depends on the size of the K8s cluster. It surely is stable but maybe we should give numbers on the size of the infrastructure. What do you think ?

File: blog/k8s-probe.md [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+post I present the probes' capabilities and potential use cases.
```
Comment (lebauce) [inline blog/k8s-probe.md]: s/probes'/probes/

File: blog/k8s-probe.md [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+metadata:
+  name: deny
+spec:
+  podSelector: {}
+  policyTypes:
+  - Egress
+```
+
+Tools such as `kubectl` extract the declarative configuration 
+(plus extra status fields) which is fine, but lacking in  terms of
```
Comment (lebauce) [inline blog/k8s-probe.md]: nit: remove double space

File: blog/k8s-probe.md [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+- pod-selectors
+- namespace-selectors
+- ip-blocks
+
+The Skydive k8s probe is able to understand all these relations and construct
+graph edges enabling to more easily reason about your k8s landscape.
+
+## Getting Started
+
+If you don't already have a k8s cluster you can setup a minimal single node k8s
```
Comment (lebauce) [inline blog/k8s-probe.md]: If you don't already have a Kubernetes cluster yet, you can setup a minimal single node cluster for evaluation purposes using `minikube`

**Review by lebauce — COMMENTED** [review]
Really useful article, thank you for this.

There are almost 60 times the term `k8s` in the blog post. I'm not suggesting to replace them with `Kubernetes` :-) But in some cases, we could just remove the word.

File: blog/k8s-probe.md [hunk 7]
```diff
@@ -0,0 +1,305 @@
...
+kubectl delete secret mysql-pass
+kubectl create secret generic mysql-pass --from-literal password=abc456
+```
+
+So we re-created `secret:mysql-pass` with **abc456** but the dependant pods
+were not re-created, and are still using the old password **abc123**.
+
+**Enter Skydive REST API.**
+
+Lets' first install the skydive binary:
```
Comment (lebauce) [inline blog/k8s-probe.md]: s/skydive/Skydive/

File: blog/k8s-probe.md [hunk 8]
```diff
@@ -0,0 +1,305 @@
...
+
+```
+kubectl expose deployment skydive-analyzer --type=LoadBalancer --port=8082
+```
+You can now open your browser at `http://localhost:8082` and view the Skydive
+dashboard.
+
+## Hello World
+
+To get a taste of the probe lets' explore the k8s hello world sample found
```
Comment (lebauce) [inline blog/k8s-probe.md]: s/lets'/let's/

File: blog/k8s-probe.md [hunk 9]
```diff
@@ -0,0 +1,305 @@
...
+installing `kubectl` can be found at
+[install-kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+Now you can setup Skydive on the k8s cluster using `kubectl apply`:
+
+```
+kubectl apply -f https://github.com/skydive-project/skydive/blob/master/contrib/kubernetes/skydive.yaml
+```
+Resulting in the deployment of:
+- skydive agent (one per node): collecting host/node information via node
```
Comment (lebauce) [inline blog/k8s-probe.md]: s/skydive/Skydive/

File: blog/index.md (hunk 1 above)
Comment (hunchback) [reply blog/index.md]: done

File: blog/index.md (hunk 2 above)
Comment (hunchback) [reply blog/index.md]: done

File: blog/k8s-probe.md (hunk 3 above)
Comment (hunchback) [reply blog/k8s-probe.md]: I agree currently put 1000 pods (based on experience I have had on 100 pod landscapes), but I  should probably add some k8s scale tests in future.

File: blog/k8s-probe.md (hunk 4 above)
Comment (hunchback) [reply blog/k8s-probe.md]: done

File: blog/k8s-probe.md (hunk 5 above)
Comment (hunchback) [reply blog/k8s-probe.md]: done

File: blog/k8s-probe.md (hunk 6 above)
Comment (hunchback) [reply blog/k8s-probe.md]: done

File: blog/k8s-probe.md (hunk 7 above)
Comment (hunchback) [reply blog/k8s-probe.md]: done

File: blog/k8s-probe.md (hunk 8 above)
Comment (hunchback) [reply blog/k8s-probe.md]: done

File: blog/k8s-probe.md (hunk 9 above)
Comment (hunchback) [reply blog/k8s-probe.md]: done

File: blog/k8s-probe.md [hunk 10]
```diff
@@ -0,0 +1,308 @@
...
+	jq '.[] | .Metadata.Name' | \
+	xargs -n 1 kubectl delete pod
+```
+
+The following figure captures the state of the system right after deleting the
+old pods (in WHITE before disappearing) by our script, and after the deployment
+resources automatically creates the new pods (in RED as they have not yet
+stabilized):
+
+<p>
```
Comment (lebauce) [inline blog/k8s-probe.md]: `<p class="center">`

File: blog/k8s-probe.md [hunk 11]
```diff
@@ -0,0 +1,308 @@
...
+  <a href="/assets/images/blog/k8s-secret.png" data-lightbox="secret" data-title="Skydive WebUI: Kubernetes Secret">
+    <img src="/assets/images/blog/k8s-secret.png"/ style="width:50%;"/>
```
Comment (lebauce) [inline blog/k8s-probe.md]: The screenshot is pretty blurry. If you still have the original, could you please increase the quality ?

Comment (lebauce) [inline blog/k8s-probe.md]: Please remove the wrong "/"

=>

Comment (hunchback) [reply blog/k8s-probe.md]: ok

Comment (hunchback) [reply blog/k8s-probe.md]: tried to improve (it was captured from the video)

File: blog/k8s-probe.md (hunk 10 above)
Comment (hunchback) [reply blog/k8s-probe.md]: ?

Comment (lebauce) [reply blog/k8s-probe.md]: I fixed the command (was HTML code)

Comment (hunchback) [reply blog/k8s-probe.md]: ok - fixed

**Comment by hunchback** [comment]
@lebauce - ready for final review/merge

**Pull request merged**